.github/actions/docker-build-image/action.yml

```
@@ -209,3 +209,16 @@ runs:
         labels: ${{ steps.meta.outputs.labels }}
         push: true
         provenance: false
+
+    - name: Print tags
+      if: env.PUSH_IMAGE == 'true'
+      shell: bash
+      run: |
+        {
+          echo "### :rocket: Docker image tags"
+          echo "| Arg                 | Value |"
+          echo "| ------------------- | ----- |"
+          echo "| Quay                | [quay.io/alfresco/${{ env.IMAGE_NAME }}](quay.io/alfresco/${{ env.IMAGE_NAME }}) |"
+          echo "| GHCR                | [ghcr.io/alfresco/${{ env.IMAGE_NAME }}](ghcr.io/alfresco/${{ env.IMAGE_NAME }}) |"
+          echo "| AWS ECR             | [${{ steps.login-ecr.outputs.registry }}/${{ env.IMAGE_NAME }}](${{ steps.login-ecr.outputs.registry }}/${{ env.IMAGE_NAME }}) |"
```

Looking at the test run at https://github.com/Alfresco/alfresco-identity-adapter-service/actions/runs/9564070539/attempts/1#summary-26364116425, the links do not actually work well (for instance the first one gives https://github.com/Alfresco/alfresco-identity-adapter-service/actions/runs/quay.io/alfresco/alfresco-identity-adapter-service:feature_AAE-23198-action-test): is giving the link actually useful? maybe an easy way to copy/paste it to a "docker pull" command would be better (?)

Also the table might look nice on adapter-service, where there's only one docker image built, but I think it should be much more compact to accommodate the process-services use case (34 images right now). Maybe we can have a try like this and see how it goes, though.

I would put the image name in the header, as suggested below (not sure if the image name in markdown title would look ok though)
If you think it's interesting to keep the links please fix them, maybe adding https:// in front is enough (and also "Arg" should be replaced by a meaningful column title)

```suggestion
          echo "### :rocket: Docker image built: \'${{ env.IMAGE_NAME }}\'"
```

---

My primary goal for tags was making it copyable, from my browser I can right click and copy (then paste it in rancher). Tried with different markdown options by probably links are the only way.